backend/Origam.Schema.DeploymentModel.UI/Origam.Schema.DeploymentModel.UI.csproj

```diff
@@ -97,6 +97,10 @@
       <Project>{976F3ABA-938D-4242-A93B-7D7A1861A66E}</Project>
       <Name>Origam.Schema</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Origam.UI.Common\Origam.UI.Common.csproj">
```

Comment:
Why was this necessary?

Reply:
Yes. The project did not want to compile without it